test/Core.Test/Services/OrganizationServiceTests.cs

```diff
@@ -649,6 +648,78 @@ await sutProvider.GetDependency<IMailService>().Received(1)
         await sutProvider.Sut.SaveUserAsync(newUserData, savingUser.UserId, collections, groups);
     }
 
+    [Theory, BitAutoData]
+    public async Task SaveUser_WithCustomPermission_WhenCustomSavingUserHasPermission_Passes(
```

Comment:
Slightly easier to read:

```suggestion
    public async Task SaveUser_WithCustomPermission_WhenSavingUserHasCustomPermission_Passes(
```